src/main/java/seedu/address/logic/parser/AddPairCommandParser.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ELDERLY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VOLUNTEER;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.AddPairCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.pair.Pair;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Parses input arguments and creates a new AddPairCommand object
+ */
+public class AddPairCommandParser implements Parser<AddPairCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddPairCommand
+     * and returns an AddPairCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddPairCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_ELDERLY, PREFIX_VOLUNTEER);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_ELDERLY, PREFIX_VOLUNTEER)
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddPairCommand.MESSAGE_USAGE));
+        }
+
+        Name elderly = ParserUtil.parseName(argMultimap.getValue(PREFIX_ELDERLY).get());
+        Name volunteer = ParserUtil.parseName(argMultimap.getValue(PREFIX_VOLUNTEER).get());
+        Phone phone = ParserUtil.parsePhone("88888888");
+        Email email = ParserUtil.parseEmail("dummy@email.com");
+        Address address = ParserUtil.parseAddress("dummy address");
+        Set<Tag> tagList = ParserUtil.parseTags(new HashSet<>());
```

Comment:
is this needed? or for testing? 
```suggestion
        Phone phone = ParserUtil.parsePhone("88888888");
        Email email = ParserUtil.parseEmail("dummy@email.com");
        Address address = ParserUtil.parseAddress("dummy address");
        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
```

Reply:
This is a placeholder, will replace it with a call to getElderly or something once the relevant classes are pulled